doc/release-notes/5.1-release-notes.md

```diff
@@ -0,0 +1,100 @@
+
+### Documentation for Purging the Ingest Queue
+
+At times, it may be necessary to cancel long-running Ingest jobs in the interest of system stability. The Troubleshooting section of the [Admin Guide](http://guides.dataverse.org/en/5.1/admin/) now has specific steps.
+
+### Biomedical Metadata Block Updated
+
+The Life Science Metadata block (biomedical.tsv) was updated.  "Other Design Type", "Other Factor Type", "Other Technology Type", "Other Technology Platform" boxes were added. See the "Additional Upgrade Steps" below if you use this in your installation.
+
+## Notes for Tool Developers and Integrators
+
+### Spaces in File Names
+
+Dataverse Installations using S3 storage will no longer replace spaces in file names of downloaded files with the + character. If your tool or integration has any special handling around this, you may need to make further adjustments to maintain backwards compatibility while also supporting Dataverse installations on 5.1+.
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.1 Milestone](https://github.com/IQSS/dataverse/milestone/90?closed=1) in Github.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](http://guides.dataverse.org/en/5.1/installation/)
+
+## Upgrade Instructions
+
+0. These instructions assume that you've already successfully upgraded from Dataverse 4.x to  Dataverse 5 following the instructions in the [Dataverse 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0).
+
+1. Undeploy the previous version.
+
+<payara install path>/payara/bin/asadmin list-applications
+<payara install path>/payara/bin/asadmin undeploy dataverse
+
+2. Stop payara and remove the generated directory, start.
+
+- service payara stop
+- remove the generated directory: rm -rf <payara install path>payara/payara/domains/domain1/generated
+- service payara start
+
+3. Deploy this version.
+<payara install path>/payara/bin/asadmin deploy <path>dataverse-5.1.war
+
+4. Restart payara
+
+### Additional Upgrade Steps
+
+1. Update Biomedical Metadata Block (if used)
+
+   `wget https://github.com/IQSS/dataverse/releases/download/5.1/biomedical.tsv`
+   `curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @biomedical.tsv -H "Content-type: text/tab-separated-values"`
+
+2. Updated Solr XML and Reload Solr
+
+-  copy schema_dv_mdb_fields.xml and schema_dv_mdb_copies.xml to solr server, for example into /usr/local/solr/solr-7.7.2/server/solr/collection1/conf/ directory
+-  reload Solr, for example, http://localhost:8983/solr/admin/cores?action=RELOAD&core=collection1
+
+3. (Recommended) Run ReExportall to update JSON Exports  
```

Comment:
Same here - none of the PRs changes the export formats do they? Neither of these hurt if you do them when not needed, but if it really is simpler, it would be nice to show that, esp. after 5.0.

Reply:
I thought if we were updating a metadata block we recommended a JSON reexport to capture those changes? 

re: formatting, I collapsed this into a single step in https://github.com/IQSS/dataverse/pull/7283/commits/af09c9a5cf771731e3bc09c9504fcb4b94bf2c04

Reply:
Ah yeah - if you update a block where there's existing metadata, you could be changing the title, or the formal URI associated with it, which would mean exports like JSON and OAI-ORE could change.